### PR TITLE
TCF-5 Map

### PR DESCRIPTION
**Pull request by flori950**

### CODE REVIEW

**Review by thdrbrkw — CHANGES_REQUESTED** [review]
Can you please remove Shrek

File: frontend/src/app/shared/map/map.component.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  selector: 'app-map',
+  templateUrl: './map.component.html',
+  styleUrls: ['./map.component.scss']
+})
+export class MapComponent implements OnInit {
+  map!: Map;
+  vectorSource!: VectorSource;
+  vectorLayer!: VectorLayer<any>;
+  overlay!: Overlay;
+
```
Comment (thdrbrkw) [inline frontend/src/app/shared/map/map.component.ts:25]: Here an `EventEmitter` is missing which emits the value of the clicked coordinates

File: frontend/src/app/shared/map/map.component.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div id="map" style="width: 100%; height: 400px;"></div>
```
Comment (thdrbrkw) [inline frontend/src/app/shared/map/map.component.html:1]: `height: 100%`

**Comment by thdrbrkw** [comment]
Committed changes:
- made map 100% for width and height
- using point instead of logo/png
- only one point displayed
- changed displayed coordinate value to long/lat

**Review by thdrbrkw — APPROVED** [review]

**Review by guttenberger — APPROVED** [review]

**Pull request merged**